tests/test_network.py

```diff
@@ -161,9 +161,14 @@ def test_network_nfs_io(env, target, shell):
             if "rw" in fs_info["options"].split(","):
                 writeable.add(ptx_work)
 
-    assert missing == set(), "These ptx-works do not have corresponding automount-units"
-    assert ptx_works == readable, "Mounts are not readable"
-    assert writeable == set(), "Mounts are writeable but should not be"
+        break
```

Comment:
This seems to be an unrelated change.

Reply:
Thanks for catching that! Removed that `break`.